### PR TITLE
add support for expression evaluation and an HTML example (ref #3)

### PR DESCRIPTION
**Pull request by davecap**

### CODE REVIEW

File: src/resource-managers/expression-manager.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ExpressionManager.prototype = Object.create(ResourceManager.prototype);
+
+/**
+ * Evaluate an expression.
+ *
+ * @param {Object} data (optional) Variables referenced by the expression.
+ * @returns {Promise} Expression API response.
+ */
+ExpressionManager.prototype.evaluate = function(data) {
+  console.log(this._solveBio);
```
Comment (jercoh) [inline src/resource-managers/expression-manager.js]: remove `console.log`

File: src/resource-managers/expression-manager.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * Class representing an Expression manager.
+ * Expressions are formulas that execute on the SolveBio API. They return a
+ * single value or list of values with a specific data type.
+ *
+ * @constructor
+ * @augments ResourceManager
+ * @requires module:resource-managers/resource-manager
+ */
+
+var ExpressionManager = function(solveBio, expression, dataType, isList) {
```
Comment (jercoh) [inline src/resource-managers/expression-manager.js:22]: Do we need the `dataType` or `isList` on instantiation?
I would have put `expression` and `data` which are parameters of the expression.
`dataType` or `isList` are more evaluation parameters.
Then we would use the following syntax:
```js
var expression = SolveBio.Expression(my_expression, my_data);
expression.evaluate(dataType, isList);
```

**Review by jercoh — CHANGES_REQUESTED** [review]
we should also have a test suite for the `ExpressionManager` class.

File: example/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<html>
```
Comment (jercoh) [inline example/index.html]: Can you please add a section in the `README.md` on how to run this example?

**Review by jercoh — CHANGES_REQUESTED** [review]

File: src/resource-managers/expression-manager.js (hunk 2 above)
Comment (davecap) [reply src/resource-managers/expression-manager.js:22]: Actually, in a sense all the parameters are used in evaluation. Would it make more sense to just not use a resource for the base of this?

Comment (jercoh) [reply src/resource-managers/expression-manager.js:22]: Yeah we can just use an `evaluationHelper`. The `SolveBio` context will have an `evaluate` method.

**Review by jercoh — APPROVED** [review]

**Pull request merged**